DigitalExcellence/dex-frontend
# Bugfix/fix overview

## Description

fixed the lazyloading of the overview and missing error image

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to https://staging.dex.software/project/overview
2. See that it takes a few seconds before the projects show up in the grid overview.
3. Review your chrome dev console and see the 404 error

## Link to issue
#374 & #372 

Closes: #374 #372
